### PR TITLE
AKU-195: Add support for inline edit properties

### PR DESCRIPTION
**Pull request by draperd**
This PR addresses https://issues.alfresco.com/jira/browse/AKU-195

### CODE REVIEW

File: aikau/src/main/resources/alfresco/services/ActionService.js [hunk 1]
```diff
@@ -579,60 +576,33 @@ define(["dojo/_base/declare",
...
-      /**
-       * Handles requests to display the details of the supplied document. This function currently
-       * delegates handling of the request to the Alfresco.DocLibToolbar by calling
-       * [callLegacyActionHandler]{@link module:alfresco/services/ActionsService#callLegacyActionHandler}
+       * Handles requests to edit the basic metadata of the node provided.
        *
        * @instance
        * @param {object} payload The response from the request
-       * @param {object} document The document to get the details for
+       * @param {object} document The node to get the details for
```
Comment (davidcognite) [inline aikau/src/main/resources/alfresco/services/ActionService.js]: should read: "@param {object} node The node to get the details for"?

File: aikau/src/main/resources/alfresco/services/ActionService.js [hunk 2]
```diff
@@ -1078,34 +970,6 @@ define(["dojo/_base/declare",
...
-         var node = lang.clone(this._currentNode.parent);
-         var record = {
-            nodeRef: node.nodeRef,
-            displayName: node.properties["cm:name"],
-            jsNode: new JsNode(node)
-         };
-         this.callLegacyActionHandler("onActionCloudUnsync", record);
-      },
-
-      /**
```
Comment (davidcognite) [inline aikau/src/main/resources/alfresco/services/ActionService.js]: I assume you intentionally deleted these unrelated methods?

File: aikau/src/test/resources/alfresco/services/ContentServiceTest.js [hunk 3]
```diff
@@ -99,13 +103,84 @@ define(["intern!object",
...
+      },
+
+      "Save basic metadata": function() {
+         return browser.findByCssSelector("#ALF_BASIC_METADATA_DIALOG .confirmationButton > span")
+            .click()
+         .end()
+         .findByCssSelector("tr.mx-row:nth-child(3) .mx-url")
+            .getVisibleText()
+            .then(function(text) {
+               assert(text.indexOf("aikau/proxy/alfresco/api/node/workspace/SpacesStore/1a0b110f-1e09-4ca2-b367-fe25e4964a4e/formprocessor") !== -1, "Update POST did not include nodeRef: " + text);
```
Comment (davidcognite) [inline aikau/src/test/resources/alfresco/services/ContentServiceTest.js]: Would assert.include be cleaner here? (http://chaijs.com/api/assert/#include-section)

Comment (draperd) [inline aikau/src/test/resources/alfresco/services/ContentServiceTest.js]: Yes, didn't know that existed - I'll update - ta :)

File: aikau/src/main/resources/alfresco/services/ActionService.js (hunk 2 above)
Comment (draperd) [inline aikau/src/main/resources/alfresco/services/ActionService.js]: Yes, we're not using the legacy action code - they wouldn't have worked anyway. We have JIRA issues to address these actions properly in the future

Comment (davidcognite) [inline aikau/src/main/resources/alfresco/services/ActionService.js]: Cool - I thought that would be the case.

**Pull request merged**